Magic/src/main/java/com/elmakers/mine/bukkit/spell/BaseSpell.java

```diff
@@ -82,7 +56,7 @@
     protected static final int MIN_Y = 1;
 
     // TODO: Configurable default? this does look cool, though.
-    protected final static Material DEFAULT_EFFECT_MATERIAL = Material.WATER;
+    protected final static Material DEFAULT_EFFECT_MATERIAL = Material.STATIONARY_WATER;
```

Comment:
Why?

Reply:
Not sure what happened there, I dont even remember changing it